src/app/services/transcription/transcription.service.ts

```diff
@@ -17,6 +19,47 @@ export const COMPLETED_TRANSCRIPTION_STATUS_ID = 6;
 export class TranscriptionService {
   constructor(private http: HttpClient) {}
 
+  // Defined in seconds
+  private readonly POLL_INTERVAL = 60;
+  private readonly TRANSCRIPT_REQUEST_COUNT_POLL_INTERVAL = 30;
```

Comment:
It's preferable to name the variables like this than to include a comment
```suggestion
  private readonly POLL_INTERVAL_SECS = 60;
  private readonly TRANSCRIPT_REQUEST_COUNT_POLL_INTERVAL_SECS = 30;
```

Reply:
Love this suggestion, changes incoming with next commit